### DISCUSSION TITLE
RasPi Version?

### DISCUSSION THREAD

**Discussion by Atronicks** [Flam3rboy/discord-bot-client#240] [Ideas]
Title says it all, i just want a version i can run from a 32-bit install of raspian.

**Comment by samuelscheit** [comment]
if someone can compile electron to 32bit it would be possible

**Reply by Atronicks** [reply]
If you can find a way then please upload it because my goal for this is a separate mini computer just for discord (to make a more seamless experience between programming my bot and testing my bot). I've already got my hands on a 32-bit version of the normal discord client, I've managed to get a microphone working along with a headset/headphones. the last thing i need is a 32-bit ver of this.
